doc/Learn/Maui/MauiOverview.md

```diff
@@ -47,7 +52,30 @@ dotnet new unoapp -preset blank -maui -o SimpleMauiApp
 
 ## Get Started - Existing Uno Application
 
-.NET MAUI Embedding can be added to any existing Uno application via the following steps. The .NET MAUI Embedding feature is only supported in Uno applications that target iOS, Android, MacCatalyst, and Windows, as shown in the Platforms folder in the following solution structure. These instructions are appropriate for an Uno application targeting .NET 7.
+.NET MAUI Embedding can be added to any existing Uno application via the following steps. The .NET MAUI Embedding feature is only supported in Uno applications that target iOS, Android, MacCatalyst, and Windows, as shown in the Platforms folder in the following solution structure.
+
+### MauiEmbedding Dependencies
+
+#### [**Single Project Template**](#tab/single-project)
+
+![Starting solution structure - SingleProject](Assets/GettingStartedExisting01-StartSingleProjectStructure.png)
+
+##### UnoFeatures
+
+In the **.csproj** file, find the `<UnoFeatures>` property and add `MauiEmbedding`. This step adds all the dependencies you need for you to have Maui Embedding working with your project.
+
+```diff
+<UnoFeatures>
+  ...
+  Serialization;
+  Localization;
+  Navigation;
+  ExtensionsCore;
```

Comment:
Why are these features listed here? None have them are required to use MauiEmbedding